python_http_client/client.py

```diff
@@ -135,7 +137,10 @@ def _make_request(self, opener, request):
         :type request: urllib.Request object
         :return: urllib response
         """
-        return opener.open(request)
+        try:
+            return opener.open(request)
+        except HTTPError as err:
+            handle_error(err)
```

Comment:
@w- Here, I am handling the urllib.HTTPError.  What you are saying is, we do this:-
```
try:
    return opener.open(request)
except HTTPError as err:
    raise get_HTTPError(err)
```
Am I right?